app-frontend/src/app/services/projects/project.service.js

```diff
@@ -410,7 +388,7 @@ export default app => {
                         visibility: params.visibility || 'PRIVATE',
                         tileVisibility: params.tileVisibility || 'PRIVATE',
                         tags: params.tags || [],
-                        isAOIProject: params.isAOIProject || false
+                        isAOIProject: false
```

Comment:
I take it we're leaving api / batch jobs for another day?

Reply:
Uhh I can do that now, it's pretty easy to do safely

Reply:
I left in the stuff on projects to avoid messing with the projects datamodel / table for now but veerywhere else it's gone